components/menu/test/menu-checkbox.visual-diff.js

```diff
@@ -7,6 +7,22 @@ describe('d2l-menu checkbox', () => {
 
 	let browser, page;
 
+	const contentResize = (page, selector) => {
```

Comment:
Beyond the scope of this PR, but if you wanted to, you _could_ simply this code a little by:
``` javascript
const { oneEvent } = require('@brightspace-ui/visual-diff/helpers');

const click = async(page, selector) => {
    const resize = oneEvent(page, selector, 'd2l-menu-item-change');
    await page.$eval(selector, (item) => item.click());
    return resize;
};
```
I think the `await` in there isn't really doing anything, in which case that could be removed, and the function itself would not need to be `async`.

The `oneEvent` helper is new - I recently did a little refactor to the visual-diff library and added that as a helper, since waiting for a single event is a common pattern.

Reply:
Updated this one and the `menu-radio` tests!  Will leave the other two for a more focused effort